singularity-eos/eos/eos_variant.hpp

```diff
@@ -223,12 +223,333 @@ class Variant {
   void DensityEnergyFromPressureTemperature(const Real press, const Real temp,
                                             Real *lambda, Real &rho, Real &sie) const {
     return mpark::visit(
-        [&press, &temp, &lambda, &rho, &sie](const auto &eos) {
+        [&press, &temp, lambda, &rho, &sie](const auto &eos) {
```

Comment:
This change should probably be reverted to be consistent with the other lambdas.

Reply:
Fixed